compiler/AST/build.cpp

```diff
@@ -639,7 +639,7 @@ buildLabelStmt(const char* name, Expr* stmt) {
         loop->userLabel = astr(name);
       }
     } else {
-      USR_FATAL(stmt, "cannot label non-loop statement");
+      USR_FATAL(stmt, "can label only for-, while-do- and do-while-statements");
```

Comment:
It might just be me, but would this scan better as "can only label..."?  (@gbtitus, did you say you proposed the current wording?)

Reply:
> It might just be me, but would this scan better as "can only label..."? (@gbtitus, did you say you proposed the current wording?)

I did say that; my proposal is in the comment [here](https://github.com/chapel-lang/chapel/issues/8914#issuecomment-470179898) in the issue this PR addresses.  But I must have scanned the new message too fast; it does indeed flip my original "can only label" to "can label only".

Reply:
@gbtitus @bradcray shall I update the error message?

Reply:
I think so, unless there's something I'm missing.

Reply:
I updated it.